src/raftstore/store/worker/pd.rs

```diff
@@ -176,6 +184,50 @@ impl<T: PdClient> Runner<T> {
             error!("report split failed {:?}", e);
         }
     }
+
+    fn handle_get_region(&self, local_region: metapb::Region, peer: metapb::Peer) {
+        metric_incr!("pd.get_region");
+        match self.pd_client.get_region(local_region.get_start_key()) {
```

Comment:
I think if the acquired Region's range can cover the search key, we can use this Region to check.